docs/en/02_Developer_Guides/05_Extending/How_Tos/02_Create_a_Google_Maps_Shortcode.md

```diff
@@ -20,7 +20,8 @@ use SilverStripe\View\Parsers\ShortcodeParser;
  
 ShortcodeParser::get('default')->register('googlemap', function($arguments, $address, $parser, $shortcode) {
     $iframeUrl = sprintf(
-        'http://maps.google.com/maps?q=%s&amp;hnear=%s&amp;ie=UTF8&hq=&amp;t=m&amp;z=14&amp;output=embed',
+        '%s://maps.google.com/maps?q=%s&amp;hnear=%s&amp;ie=UTF8&hq=&amp;t=m&amp;z=14&amp;output=embed',
+        (isset($_SERVER['HTTPS'])) ? 'https': 'http',
```

Comment:
Maybe you could just use `//maps.goo...`?

Reply:
You're right. I did not think about that :)